src/nodes.coffee

```diff
@@ -706,19 +713,21 @@ exports.Call = class Call extends Base
   # splatArgs is an array of CodeFragments to put into the 'apply'.
   compileSplat: (o, splatArgs) ->
     if this instanceof SuperCall
-      return [].concat @makeCode("#{ @superReference o }.apply(#{@superThis(o)}, "),
-        splatArgs, @makeCode(")")
+      reference = @superReference o
+      if reference is 'super' or reference.indexOf('super.') is 0
```

Comment:
Uses ES2015 splats in `super` calls. Hopefully these changes will be superseded by a change to ES2015 splats.